marlowe-chain-sync/libchainsync/Language/Marlowe/Runtime/ChainSync.hs

```diff
@@ -1,41 +1,45 @@
 {-# LANGUAGE Arrows #-}
```

Comment:
Changes in this file:

- Moving otel rendering from Logging.hs
- Propagation of Connection refactoring
- Moving server sources to component output.